### PR TITLE
[T2A2][W10-A1]Karen Ang

### PR DESCRIPTION
**Pull request by karen**

### CODE REVIEW

**Comment by mauris** [comment]
hey @karenang, do you have a team (A1, A2, A3 or A4) yet?

**Comment by karen** [comment]
Hi @mauris, I've updated the title to reflect the group number.

**Comment by mauris** [comment]
@karenang nice thanks :D

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -767,17 +767,6 @@ private static void addPersonToAddressBook(String[] person) {
     }
 
     /**
-     * Deletes a person from the address book, target is identified by it's absolute index in the full list.
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/AddressBook.java:769]: Good observation here.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1046,7 +1048,8 @@ private static boolean isPersonNameValid(String name) {
      * @return whether arg is a valid person phone number
      */
     private static boolean isPersonPhoneValid(String phone) {
-        return phone.matches("\\d+");    // phone nonempty sequence of digits
+    	final String NONEMPTYDIGITS = "\\d+";
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/AddressBook.java:1040]: Please check java naming convention to see how to separate two words in naming of final strings. It is good to rename regex both for readability and better maintenance later.

**Comment by OuyangDanwen** [comment]
Generally well done!

**Pull request closed without merging**